apps/pie-docs/src/components/tag/overview/overview.md

```diff
@@ -44,315 +47,332 @@ Tags can be embedded in other components such as cards, data tables (among other
 {% contentPageImage {
     src:"../../../assets/img/components/tag/anatomy.svg",
     alt: "Anatomy of a tag.",
-    width: 120
+    width: 169
 } %}
 
 {% list {
     type: listTypes.ordered,
     items: [
         "**Icon (Optional):** Visually supports the label.",
-        "**Label:** Provides information about the content or purpose of the tag.",
-        "**Container**: Background container that organises the information."
+        "**Label:** Provides informative information to the user.",
+        "**Trailing icon (Optional):** Indicates additional actions or further interactions."
```

Comment:
Just to check we haven't missed a requirement here - does the trailing icon emit a different event to the leading?